### PR TITLE
CSIMigrationvSphere doc update

### PR DESCRIPTION
**Pull request by divyenpatel**
Documentation update for `CSIMigrationvSphere` feature gate.

KEP - https://github.com/kubernetes/enhancements/issues/1491

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -79,7 +79,8 @@ different Kubernetes components.
 | `CSIMigrationRBD` | `false` | Alpha | 1.23 | |
 | `CSIMigrationvSphere` | `false` | Alpha | 1.18 | 1.18 |
 | `CSIMigrationvSphere` | `false` | Beta | 1.19 | 1.24 |
-| `CSIMigrationvSphere` | `true` | Beta | 1.25 | |
+| `CSIMigrationvSphere` | `true` | Beta | 1.25 | 1.25 |
+| `CSIMigrationvSphere` | `true`  | GA | 1.26  | -  |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:83]: These lines are supposed to be migrated to the second table in the page, which is for GA'ed or deprecated features.

File: content/en/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -921,20 +921,15 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
...
 
 [vSphere CSI driver](https://github.com/kubernetes-sigs/vsphere-csi-driver)
 must be installed on the cluster. You can find additional advice on how to migrate in-tree `vsphereVolume` in VMware's documentation page 
 [Migrating In-Tree vSphere Volumes to vSphere Container Storage Plug-in](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-968D421F-D464-4E22-8127-6CB9FF54423F.html).
 
-As of Kubernetes v1.25, vSphere releases less than 7.0u2 are not supported for the
-(deprecated) in-tree vSphere storage driver. You must run vSphere 7.0u2 or later
-in order to either continue using the deprecated driver, or to migrate to
-the replacement CSI driver.
+You must run vSphere 7.0u2 or later in order to migrate to the vSpehre CSI driver.
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md:932]:
```suggestion
You must run vSphere 7.0u2 or later in order to migrate to the vSphere CSI driver.
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (divyenpatel) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: done.

**Comment by divyenpatel** [comment]
@sftim @tengqm  I have addressed your review comments.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -271,6 +271,8 @@ different Kubernetes components.
 | `CSIMigrationOpenStack` | `true` | GA | 1.24 | |
 | `CSIMigrationOpenStackComplete` | `false` | Alpha | 1.17 | 1.20 |
 | `CSIMigrationOpenStackComplete` | - | Deprecated | 1.21 | - |
+| `CSIMigrationvSphere` | `true` | Beta | 1.25 | 1.25 |
+| `CSIMigrationvSphere` | `true`  | GA | 1.26  | -  |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:275]: These move from the previous table (scroll up)
```suggestion
| `CSIMigrationvSphere` | `false` | Alpha | 1.18 | 1.18 |
| `CSIMigrationvSphere` | `false` | Beta | 1.19 | 1.24 |
| `CSIMigrationvSphere` | `true` | Beta | 1.25 | 1.25 |
| `CSIMigrationvSphere` | `true` | GA | 1.26 | - |
```

File: content/en/docs/concepts/storage/volumes.md [hunk 4]
```diff
@@ -921,20 +921,15 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 #### vSphere CSI migration {#vsphere-csi-migration}
 
-{{< feature-state for_k8s_version="v1.19" state="beta" >}}
-
-The `CSIMigrationvSphere` feature for `vsphereVolume` is enabled by default as of Kubernetes v1.25.
-All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver unless `CSIMigrationvSphere` feature gate is disabled.
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
+All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver.
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md:926]:
```suggestion
In Kubernetes {{< skew currentVersion >}}, all operations for the in-tree `vsphereVolume` type
are redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver.
```
(the feature state shortcode is a strong hint about when this behavior came in).

What happens if the CSI driver isn't working / isn't installed? It'd be nice to explain that bit.

**Review by sftim — COMMENTED** [review]
Make sure to delete these lines from the feature gates page:
```
| `CSIMigrationvSphere` | `false` | Alpha | 1.18 | 1.18 |
| `CSIMigrationvSphere` | `false` | Beta | 1.19 | 1.24 |
| `CSIMigrationvSphere` | `true` | Beta | 1.25 | |

```

File: content/en/docs/concepts/storage/volumes.md (hunk 4 above)
Comment (divyenpatel) [reply content/en/docs/concepts/storage/volumes.md]: added this line 

> If vSphere CSI Driver is not installed volume operations can not be performed on the PV created with the in-tree `vsphereVolume` type.

**Pull request closed without merging**